docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: "new-referencevalue-parameter-create-payment-endpoint"
+title: "New referenceValue parameter in the Create Payment endpoint request body"
+createdAt: 2023-10-24T00:00:00.000Z
+hidden: false
+type: "added"
+excerpt: "New parameter available in the Create Payment endpoint request body."
+---
+The `referenceValue` parameter was added to the [Create Payment endpoint](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments) request body. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied.
+
+The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero.
+
+See the examples below:
+
+- **Case 1**: Order value is 15.00 (interest free):
+
+  - `value`: 15.00
+  - `referenceValue`: 15.00
+  - `installments`: 1
+  - `installmentsInterestRate`: 0.00
+
+- **Case 2**: Order value is 15.00 (with interest of 1.00 per month, 3 installments):
+
+  - `value`: 15.00
```

Comment:
This should be formatted as a request body example, not a list. If the body is too big, you can use `...` to represent the parts that don't matter to this release note.

Reply:
Corrigido.